repos/rust-lang/ci.toml

```diff
@@ -0,0 +1,7 @@
+org = "rust-lang"
+name = "ci"
```

Comment:
I'd probably call it ci-feedback then? Or ci-planning?

Reply:
sure, I renamed it ci-feedback. We can always rename it in the future if needed 👍